packages/generator/ni_measurement_plugin_sdk_generator/client/templates/measurement_plugin_client.py.mako

```diff
@@ -187,3 +241,15 @@ class ${class_name}:
 
         for response in self._get_stub().Measure(request):
             yield self._deserialize_response(response)
+
+    def register_pin_map(self, pin_map_path: pathlib.Path) -> None:
+        """Registers the pin map with the pin map service.
+
+        Args:
+            pin_map_path: Absolute path of the pin map file.
+        """
+        pin_map_id = self._get_pin_map_client().update_pin_map(pin_map_path)
+        if self._pin_map_context is None:
+            self._pin_map_context = PinMapContext(pin_map_id=pin_map_id, sites=[0])
```

Comment:
Will the corresponding LabVIEW API also default to sites=[0]?

Reply:
Currently, LabVIEW client also sets the sites to` [0]` by default.
![image](https://github.com/user-attachments/assets/005fe1a6-2829-4feb-9eb9-42c491e146bf)

In LabVIEW, users cannot set external sites like registered pin map id, but it will be supported soon.